projects/noloan2/app/src/main/res/layout/activity_lawsuit.xml

```diff
@@ -208,7 +208,7 @@
                 android:layout_height="32dp"
                 android:layout_gravity="center"
                 android:layout_margin="10dp"
-                android:text="@string/form_button_create_text"
+                android:text="@string/form_button_text_next"
```

Comment:
s/form_button_text_next/form_button_next_text ?

Reply:
I don't mind, as long as it's consistent.